docs/development_environment.mdx

```diff
@@ -130,6 +130,14 @@ Install the requirements with a provided script named `setup`.
 script/setup
 ```
 
+:::note
```

Comment:
I thought this kind of troubleshooting help should stand out more than just a regular paragraph but I wasn't sure if the `note` or `info` admonition would be more appropriate.

![image](https://github.com/user-attachments/assets/466c8ee1-1add-4ad0-b54b-60ee887cfa55)

![image](https://github.com/user-attachments/assets/4844c259-6dc6-46a0-a312-8073ca6539b6)

Let me know if there are some content or other kinds of guidelines I should follow.